packages/blip/app/components/chart/daily.js

```diff
@@ -23,8 +23,7 @@ import WindowSizeListener from 'react-window-size-listener'
 import i18next from 'i18next'
 
 import CalendarTodayIcon from '@mui/icons-material/CalendarToday'
-import InputAdornment from '@mui/material/InputAdornment'
-import TextField from '@mui/material/TextField'
+import { CircularProgress, InputAdornment, TextField } from '@mui/material'
```

Comment:
Wrong imports, need to import each component
```suggestion
import CircularProgress from '@mui/material/CircularProgress'
import InputAdornment from '@mui/material/InputAdornment'
import TextField from '@mui/material/TextField'
```